bower-locker-common.js

```diff
@@ -37,12 +37,18 @@ function getDependency(filepath) {
  */
 function getAllDependencies() {
     var bowerDependencies = fs.readdirSync('./bower_components', {encoding: 'utf8'});
-    return bowerDependencies.map(function(dirname) {
+
+    var dependencyInfos = [];
+    bowerDependencies.map(function(dirname) {
```

Comment:
`.map` is not the idiomatic choice in this case since we're not mapping values 1:1 anymore. Use `.foreach` or `.reduce` instead.